### PR TITLE
Add AccountBalance model for loading cash positions

### LINKED ISSUE #44: Support cash positions
Knowing how much "spare" cash is lying about would be handy for rebalancing, for example.

### PR DESCRIPTION
**Pull request by jspahrsummers**
Resolves #44.

@ShadoFlameX Can you look over the support I'm adding for the brokers you wrote? Not 100% positive that I interpreted the file formats correctly.

- [x] Add support to `Rebalance.ipynb`
- [x] Add command to show account balances
- [x] [Order currency pairs](https://qr.ae/TWGN7v) for conversion

### CODE REVIEW

File: bankroll/analysis.py [hunk 1]
```diff
@@ -96,24 +96,35 @@ def currencyConversionRates(
...
+         quote.market) if instrument.quoteCurrency == quoteCurrency else
+        (
+            instrument.quoteCurrency,
+            Cash(
+                currency=instrument.baseCurrency,
+                # FIXME: This unfortunately does not retain much precision when
+                # dividing by JPY in particular (where the integral portion can
+                # be quite large).
+                # See https://github.com/jspahrsummers/bankroll/issues/37.
+                quantity=Decimal(1) / quote.market.quantity))
```
Comment (jspahrsummers) [inline bankroll/analysis.py]: cc #37

File: tests/test_fidelity.py [hunk 2]
```diff
@@ -186,5 +186,15 @@ def test_securityTransferSale(self) -> None:
...
 
 
+class TestFidelityBalance(unittest.TestCase):
+    def setUp(self) -> None:
+        self.balance = fidelity.FidelityAccount(
+            positions=Path('tests/fidelity_positions.csv')).balance()
+
+    def testUSDBalance(self) -> None:
+        self.assertEqual(self.balance.cash,
+                         {Currency.USD: helpers.cashUSD(Decimal('21087.65'))})
```
Comment (ShadoFlameX) [inline tests/test_fidelity.py]: This is not the correct cash balance in the positions CSV. You're looking for `15678.89`

Comment (jspahrsummers) [reply tests/test_fidelity.py]: Will follow up in chat

**Comment by jspahrsummers** [comment]
@ShadoFlameX Thanks for reviewing! Sorry it took me a while to get back to this—was out of town. I've updated it according to the comment you left here, but I'll follow up elsewhere to make sure I'm understanding this correctly, for my own edification if nothing else.

**Pull request merged**